packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 15.0.2
+
+* Prevents optional and non-positional parameters in flutterApis.
```

Comment:
Nit: Flutter APIs.

Reply:
This didn't get changed.

Reply:
change must have gotten lost when I merged branches, I definitely changed it.